lib/structures/StoreChannel.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const GuildChannel = require("./GuildChannel");
+
+/**
+* Represents a store channel
+* @extends GuildChannel
+* @prop {String} id The ID of the channel
+* @prop {String} mention A string that mentions the channel
+* @prop {Number} type The type of the channel
+* @prop {Guild} guild The guild that owns the channel
+* @prop {String?} parentID The ID of the category this channel belongs to
+* @prop {String} name The name of the channel
+* @prop {Number} position The position of the channel
+* @prop {Boolean} nsfw Whether the channel is an NSFW channel or not
+* @prop {Collection<PermissionOverwrite>} permissionOverwrites Collection of PermissionOverwrites in this channel
+*/
+class StoreChannel extends GuildChannel {
+    constructor(data, guild) {
+        super(data, guild);
+    }
+
+    update(data) {
+        super.update(data);
```

Comment:
Redundant

Reply:
They are there for easier editing in the future.

Reply:
Add em in in the future then